Scrapper-Service/test/mongodb_test.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import json
+import warnings
+from datamodels import Conference
+from datamodels import Metadata
+from database  import mdb
+from datetime import datetime
+from logging import Logger
+
+class MongoDbTestCase(unittest.TestCase):
+    def __init__(self, methodName):
+        super().__init__(methodName)
+        self.mongo_db = mdb.MongoDatabase(Logger,"Conference_Notify","conferences")
+        
+    def cvt(self,dt):
+        dt = dt.split('+')
+        time = dt[0]+'Z'
+        return datetime.strptime(time,"%Y-%m-%dT%H:%M:%S.%fZ")
+    
+    def cleanText(self, x):
+        del x['bulkText']
+        del x['metadata']
+        del x['url']
+        return x
+    
+    def cvtDate(self,x):
+        x['deadline'] = self.cvt(x['deadline'])  
+        x['dateRange'] = [self.cvt(x['dateRange'][0]),
+                                self.cvt(x['dateRange'][1])]
+        x['finalDue'] = self.cvt(x['finalDue'])
+        x['metadata']['plugins']['wikicfp']['dateExtracted'] = self.cvt(x['metadata']['plugins']['wikicfp']['dateExtracted'])
+        print()
+        x['notificationDue'] = self.cvt(x['notificationDue'])
+        return x
+    
+    def read(self,f_name):
+        data = open(f_name,'r')
+        json_data = json.loads(data.read())
+        id_lst = []
+        for x in json_data:
+            x = self.cvtDate(x)
+            id_lst.append(x)
+        return id_lst 
+    
+    def test_data(self):
```

Comment:
* test is improper , the test does three task in one single test. A unit test should focus on only one task and one task only. Divide this in three sepparate task.